helm/cluster-cloud-director/values.schema.json

```diff
@@ -129,46 +146,62 @@
                 "ovdcNetwork",
                 "site"
             ],
+            "title": "Cloud Director",
             "type": "object"
         },
         "cloudProvider": {
             "properties": {
                 "enableVirtualServiceSharedIP": {
+                    "description": "TODO.",
+                    "title": "TODO",
                     "type": "boolean"
                 },
                 "oneArm": {
+                    "description": "TODO.",
                     "properties": {
                         "enabled": {
+                            "title": "Enable",
                             "type": "boolean"
                         }
-                    }
+                    },
+                    "title": "TODO"
```

Comment:
What is `oneArm`? Thanks for an explanation.

Reply:
oneArm defines whether to use an internal IP for the virtual service with a NAT rule to expose the external IP (true) or to expose the virtual service directly with the external IP (false).
Better explained [here](https://github.com/giantswarm/cloud-provider-cloud-director-app#notes-on-enablevirtualservicesharedip).